sosivio/README.md

```diff
@@ -0,0 +1,33 @@
+# Sosivio
+ 
+## Overview
+ 
+Sosivio is a Predictive Troubleshooting platform built specifically for Kubernetes environments and applications. Sosivio provides predictive troubleshooting tools, root cause determination, and instant remediation for any failures in Kubernetes environments. Instantly start observing all flavors of Kubernetes with real-time metrics and cluster health checks, and use lean AI to predict and prevent critical failures before they appear.
+
+Sosivio's Datadog integration allows users to see Sosivio Failure Alerts directly in Datadog dashboards and instantly be redirected to the Sosivio UI to remediate those failures (with Sosivio Premium license). You can also add context around Datadog's signals with Sosivio's root cause determination for critical failures.
+ 
+## Setup
+
+If you don't have a Sosivio account set up, [create an account][5] and sign up for a free 4-week trial of Sosivio Premium directly from our website (no credit card required). At the end of the 4-week trial, your license converts to the Sosivio Community Version, which is free, forever. With the account set up, you are ready to start using the Sosivio Application in Datadog.
+
+
+Sosivio is installed under one namespace (labeled "sosivio") which creates all the required components for the product. 
+
+ 
+### Installation
+
+1. In the Sosivio Dashboard Configuration page, add the [Datadog API key][2] and the Datadog URL (if not the default datadoghq.com site). See [Datadog sites][4] to learn more.
```

Comment:
Maybe a hyperlink to the dashboard configuration page would be helpful here?

Reply:
Dashboard is dynamic by deployment (on-prem)